src/main/java/com/avispl/symphony/dal/avdevices/mixer/vaddio/easyip/EasyIPMixerCommunicator.java

```diff
@@ -445,21 +491,21 @@ public void controlProperty(ControllableProperty controllableProperty) throws Ex
 						indexCamera = EasyIPMixerMapping.getValueByName(group.replace(EasyIPMixerConstant.VIDEO_INPUT, EasyIPMixerConstant.EMPTY));
 						value = checkValidInput(1, 20, value);
 						command = propertyItem.getControlCommand().replace("$1", indexCamera).replace("$2", value);
-						sendCommandToControlDevice(command, value, propertyKey);
+						sendCommandToControlDeviceWithExecutor(command, value, propertyKey);
 						updateCachedDeviceData(cacheKeyAndValue, property, value);
 						break;
 					case CAMERA_PAN:
 						indexCamera = EasyIPMixerMapping.getValueByName(group.replace(EasyIPMixerConstant.VIDEO_INPUT, EasyIPMixerConstant.EMPTY));
-						value = checkValidInput(-150f, 150f, value);
+						value = checkValidInput(-156.3, 151.7, value);
```

Comment:
These limits are supposed to be this specific?

Reply:
> These limits are supposed to be this specific?

![image](https://github.com/AVISPL/dal-avdevices-mixer-vaddio-easyip/assets/90905279/a2bac070-ff27-447d-bccc-d02bebe3ff2d)
There is a difference between real device and documentation.
According to the discussion, Nicolas chose the range according to real device